siad/daemon.go

```diff
@@ -125,6 +139,7 @@ func startDaemon(config Config) (err error) {
 		if err != nil {
 			return err
 		}
+		defer g.Close()
```

Comment:
Ah okay, I was confused because the implementation of `Server` in server_helpers_test.go does close the modules.